app/views/virtual-tables/index.html

```diff
@@ -292,6 +292,7 @@ <h3>{{virtualArticle.title.en}}</h3>
             </script>
 
             <script type="text/ng-template" id="filterButtons">
+                <cbd-add-new-article ng-vue="vueOptions" v-if="filterAdminTags" :admin-tags="filterAdminTags" button-text="Add Publication/Events" class="pull-right" css-class="btn btn-default"></cbd-add-new-article> 
```

Comment:
If you implement above chnage (class + slot) 
```suggestion
                <cbd-add-new-article ng-vue="vueOptions" v-if="filterAdminTags" :admin-tags="filterAdminTags" class="pull-right" class="btn btn-default">Add Publication/Events</cbd-add-new-article> 
```

Reply:
fixed